dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -401,7 +402,7 @@ public void ContentFallbackFalse_PageFallbackTrue_PageEnglish_ViewEnglishContent
                     .setPageMode(PageMode.LIVE)
                     .build(),
                 mockRequest, mockResponse);
-        Assert.assertTrue("ENG = "+html , html.contains("content1") && html.contains("content2"));
+        assertTrue("ENG = "+html , html.contains("content1") && html.contains("content2"));
```

Comment:
Codacy found an issue: [The String literal "ENG = " appears 6 times in this file; the first occurrence is on line 405](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6309437)